### DISCUSSION TITLE
Having issue with 100MHz NO COMPRESSION , UE is not getting attach.

### DISCUSSION THREAD

**Discussion by devgran** [srsran/srsRAN_Project#702] [General Help]
We are working on srs ran project 20.04 release. We are using AMD ZCU670 O-RU design.
UE is able to attach for 10MHz, 20MHz, 30MHz, 40MHz, 50MHz, 60MHz, 70MHz as a cell
bandwidth with NO COMPRESSION method.
But when we are trying 80MHz, 90MHz, 100MHz , UEs are not getting attach.

Below url is not working log file(100MHz and NO COMPRESSION).

[https://github.com/devgran/file-share-srsran/blob/main/notworking_nocomp_100MHz.log](https://github.com/devgran/file-share-srsran/blob/main/notworking_nocomp_100MHz.log)

below url is working log file(50MHz and NO COMPRESSION)

[https://github.com/devgran/file-share-srsran/blob/main/working_nocomp_50MHz.log](https://github.com/devgran/file-share-srsran/blob/main/working_nocomp_50MHz.log)

NO COMPRESSION setting in yml file is 
compr_method_ul: none                                     # Uplink compression method.
  compr_bitwidth_ul: 16                                       # Uplink IQ samples bitwidth after compression.
  compr_method_dl: none                                      # Downlink compression method.
  compr_bitwidth_dl: 16                                     # Downlink IQ samples bitwidth after compression.
  compr_method_prach: none                                    # PRACH compression method.
  compr_bitwidth_prach: 16   

Please suggest how to make  working 100MHz bandwidth with NO COMPRESSION method?

**Comment by andrepuschmann** [comment]
I don't think the log links are working. Could you perhaps also pass a full fronthaul PCAP?

**Reply by devgran** [reply]
We solved this issue by updating latest version of SRS ran. 
Thanks for your quick reply.
